models/yiiModels/DataAnalysisAppSearch.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+//******************************************************************************
+//                         DataAnalysisAppSearch.php
+// SILEX-PHIS
+// Copyright © INRA 2019
+// Creation date: 25 Feb, 2019
+// Contact: arnaud.charleroy@inra.fr, anne.tireau@inra.fr, pascal.neveu@inra.fr
+//******************************************************************************
+
+namespace app\models\yiiModels;
+
+use Yii;
+use openSILEX\opencpuClientPHP\OpenCPUServer;
+use app\models\wsModels\WSConstants;
+use yii\helpers\Url;
+use Symfony\Component\Yaml\Yaml;
+use GuzzleHttp\Exception\ClientException;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7;
+use openSILEX\opencpuClientPHP\classes\CallStatus;
+
+include_once '../config/web_services.php';
+require_once '../config/config.php';
+
+/**
+ * DataAnalysisAppSearch search class which makes link
+ * between openSILEX and OpenCPU
+ * @author Arnaud Charleroy <arnaud.charleroy@inra.fr>
+ */
+class DataAnalysisAppSearch {
+    /**
+     *
+     * @var OpenCPUServer 
+     */
+    public $ocpuserver;
+
+    /**
+     * Configuration file constants
```

Comment:
Can you explain which configuration file is it ? You could maybe add its path ?

Reply:
The configuration file is use to integrate the form with Yii2 it has been remove from this pull request.

Reply:
Explain the configuration file content in the code, not just for me ;)